File: SarfMalzemeStok.Domain/Model/CompanyMaterial.cs
using Abp.Domain.Entities;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace SarfMalzemeStok.Domain.Model
{
    public class CompanyMaterial:Entity
    {
        public int CompanyId { get; set; }
        public int MaterialId { get; set; }
        public double BirimMaliyet { get; set; }
        public int TedarikSuresi { get; set; }
        public int AsgariPartiBuyuklugu { get; set; }

        public Material material { get; set; }
        public Company company { get; set; }

    }
}

File: SarfMalzemeStok/ClientApp/src/app/models/CompanyMaterial.ts
import { Material } from './material';
import { Company } from './company';

export class CompanyMaterial {
    id: number;
    companyId: number;
    materialId: number;
    birimMaliyet: number;
    tedarikSuresi: number;
    asgariPartiBuyuklugu: number;
    material: Material;
    company: Company;
    
}
